lib/msal-browser/src/naa/AccountInfo.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export type AccountInfo = {
+    homeAccountId: string;
+    environment: string;
+    tenantId: string;
+    username: string;
+    localAccountId: string;
+    name?: string;
+    idToken?: string; // idTokenClaims can be parsed from idToken in MSAL.js
+    platformBrokerId?: string; // Used by WAM previous called nativeAccountId
```

Comment:
Curious why you can't just use the existing nativeAccountId to save you from redefining this type?

If absolutely necessary I would suggest redefining to inherit the existing type so you get new additions out of the box

```
export type AccountInfo = Omit<CommonAccountInfo, "nativeAccountId"> & {
platformBrokerId?: string
}
```

Reply:
Isn't the "bridge model" a new thing we're defining and building? Why are we not designing the thing to use the interfaces we already have?

Reply:
native and platform is going to be confusing. I will mark it as a follow up with Taos to unify this.